lua/apg/cl_utils.lua

```diff
@@ -111,22 +111,22 @@ end
 
 function utils.numSlider( panel, x, y, w, h, text, var, min, max, decimal )
     local slider = vgui.Create( "DNumSlider", panel )
-        slider:SetPos( x, y )           // Set the position
-        slider:SetSize( w, h )      // Set the size
-        slider:SetText( "" )    // Set the text above the slider
-        slider:SetMin( min )                // Set the minimum number you can slide to
-        slider:SetMax( max )                // Set the maximum number you can slide to
-        slider:SetDecimals( decimal )           // Decimal places - zero for whole number
+        slider:SetPos( x, y )          -- Set the position

```

Comment:
I like the way this is formatted better. Some people seem to have mixed feelings about aligning comments (citing annoyance when needing to change or add lines later and then needing to re-align the comments)

Personally, I think they look good.

However, I think these comments are sort of worthless. They don't provide any more info than the actual code. The only line that might need comments are the `SetMin` and `SetMax` lines, but I'd argue that a better solution would be to change the parameter names.

e.g.

```lua
function utils.numSlider( panel, x, y, w, h, text, var, minSlide, maxSlide, decimal )
    ...
    slider:SetSize( w, h )
    slider:SetText( "" )
    slider:SetMin( minSlide )
    slider:SetMax( maxSlide )
    slider:SetDecimals( decimal )
```

This way you can see that the `SetMin` and `SetMax` methods are modifying the min/max of the sliding in some way. With context it's easy to discern its purpose without superfluous comments.

Just my soapbox about unnecessary comments. I think Lua is a language that lets you get away with almost no comments - when written well (descriptive variable/function names, using multiple steps in equations, etc.), the code is self-commenting